src/cli/utils/flags.go

```diff
@@ -167,6 +177,36 @@ func AddSiteFlag(cmd *cobra.Command) {
 		"Backup data by site URL; accepts '"+Wildcard+"' to select all sites.")
 }
 
+func AddMaintenanceSafetyFlag(cmd *cobra.Command) {
+	fs := cmd.Flags()
+	fs.StringVar(
+		&MaintenanceSafetyFV,
+		MaintenanceSafetyFN,
+		string(control.FullSafety),
+		"Disable safety measures to run maintenance faster. Improper usage can break a repository")
+	cobra.CheckErr(fs.MarkHidden(MaintenanceSafetyFN))
+}
+
+func AddQuickMaintenanceFlag(cmd *cobra.Command) {
+	fs := cmd.Flags()
+	fs.BoolVar(
+		&QuickMaintenanceFV,
+		QuickMaintenanceFN,
+		false,
+		"Run quick maintenance that won't clean up and optimize the repo as much")
+	cobra.CheckErr(fs.MarkHidden(QuickMaintenanceFN))
+}
+
+func AddForceMaintenanceFlag(cmd *cobra.Command) {
+	fs := cmd.Flags()
+	fs.BoolVar(
+		&ForceMaintenanceFV,
+		ForceMaintenanceFN,
+		false,
+		"Always run maintenance even if it may be unsafe")
```

Comment:
nit: `always run` sounds like a persistent, cron-like configuration.
```suggestion
		"Force maintenance even if it may be unsafe")
```

Reply:
`Force maintenance. Caution: This may be unsafe` ?